src/main.js

```diff
@@ -8,6 +8,9 @@
     See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 */
 
+// import * as Sentry from '@sentry/electron';
+// Sentry.init({dsn: 'https://def94db0cce14e2180e054407e551220@sentry.vircadia.dev/3'});
```

Comment:
Remove commented-out code.

Reply:
For now, I keep it there to uncomment in the parts that it is for testing builds distributed to non-technical users for feedback.